### PR TITLE
feat: Add AutoScaling ScheduledActions

### LINKED ISSUE #615: Add Support AWS::AutoScaling::ScheduledAction
AWS::AutoScaling::ScheduledAction

details can be found [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-as-scheduledaction.html)

### PR DESCRIPTION
**Pull request by spangenberg**
Fixes #615

### CODE REVIEW

File: resources/services/autoscaling/scheduled_actions.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+				Description: "The minimum size of the Auto Scaling group.",
+				Type:        schema.TypeInt,
+			},
+			{
+				Name:        "recurrence",
+				Description: "The recurring schedule for the action, in Unix cron syntax format",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "scheduled_action_arn",
```
Comment (roneli) [inline resources/services/autoscaling/scheduled_actions.go:68]: scheduled_action_arn -> arn

File: resources/services/autoscaling/scheduled_actions.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+				Description: "The Amazon Resource Name (ARN) of the scheduled action.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ScheduledActionARN"),
+			},
+			{
+				Name:        "scheduled_action_name",
```
Comment (roneli) [inline resources/services/autoscaling/scheduled_actions.go:74]: scheduled_action_name -> name

**Review by roneli — COMMENTED** [review]
Two nits, will approve once e2e tests pass.

**Comment by spangenberg** [comment]
Blocked on #703

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**